Competition2017/src/competition/subsystems/agitator/AgitatorSubsystem.java

```diff
@@ -41,30 +59,62 @@ public AgitatorSubsystem(
         this.agitatorMotor.createTelemetryProperties(side.toString(), propManager);
     }
 
-    public void setAgitatorPower(double power) {
+    protected void setAgitatorPowerRaw(double power) {
         agitatorMotor.ensureTalonControlMode(TalonControlMode.PercentVbus);
         agitatorMotor.set(power);
     }
+    
+    public void setAgitatorPower(double power) {
+        setAgitatorPowerRaw(power);
+        unjamEnd = -1;
+        isIntaking = false;
```

Comment:
My thinking was that the public interface allowed you to use the three preset powers or set your own. We don't want to deal with unjamming in eject mode or other similar oddities, so I implemented this flag which only activates the feature when the agitator was started in "intake" mode. I could remove the public setter if we don't need it.